src/test/perl/fetch.t

```diff
@@ -413,7 +414,7 @@ isa_ok($pf, "CAF::FileReader", "download retruns CAF::FileReader instance");
 my %r = $f->previous();
 like(*{$r{url}}->{filename}, qr{profile.url$},
      'Correct file read for the previous URL');
-like(*{$r{profile}}->{filename}, qr{profile.xml$},
+like(*{$r{profile}}->{filename}, qr{profile$},
```

Comment:
so the bad news is that the current code doesn't work: symonk in CAF::Path is a method (or you are importing the symlink from LC). and thus this code needs to be converted to CAF::Path (so all mkdir and chmod etc can also be handled by CAF::path).

that's overkill i think (at the very least a lot of work), so better use the regular perl symlink calls, and also a regular test if a file is a symlink (it will be a broken symlink probably in the test, because the actual profile is most likely mocked in the tests, so there's no real file)

something in between would involve using `mkcafpath` from CAF::Path, but it will be ugly.

Reply:
Damn.

Well the code in this PR actually worked on a test host…
```
# ls -l /var/lib/ccm/profile.116
total 7972
-rw------- 1 root root 3652790 Aug 24 15:49 eid2data.db
-rw------- 1 root root       9 Aug 24 15:49 eid2data.fmt
-rw------- 1 root root 2075006 Aug 24 15:49 path2eid.db
-rw------- 1 root root       9 Aug 24 15:49 path2eid.fmt
-rw------- 1 root root 1059813 Aug 24 15:49 profile
-rw------- 1 root root      70 Aug 24 15:49 profile.url
lrwxrwxrwx 1 root root      32 Aug 24 15:49 profile.xml -> /var/lib/ccm/profile.116/profile
-rw------- 1 root root 1359758 Aug 24 15:49 tabcompletion
```

Reply:
well, it works, you can test with `ok(-l $filename, "$filename is a symlink");`

Reply:
I'll try doing it with `mkcafpath`

Reply:
@jrha not sure it is worth it. use the regular p[erl `symlink` (simply remove the `use CAF::path qw(symlink)`); and test with `-l`

and maybe open an issue to port CCM to CAF::Path